opensrp-giz-malawi/src/main/java/org/smartregister/giz/model/NavigationModel.java

```diff
@@ -0,0 +1,50 @@
+package org.smartregister.giz.model;
+
+import android.util.Log;
+
+import org.smartregister.giz.R;
+import org.smartregister.giz.contract.NavigationContract;
+import org.smartregister.giz.util.GizConstants;
+import org.smartregister.util.Utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class NavigationModel implements NavigationContract.Model {
+    private static NavigationModel instance;
+    private static List<NavigationOption> navigationOptions = new ArrayList<>();
+    private String TAG = NavigationModel.class.getCanonicalName();
+
+    public static NavigationModel getInstance() {
+        if (instance == null)
+            instance = new NavigationModel();
+
+        return instance;
+    }
+
+    @Override
+    public List<NavigationOption> getNavigationItems() {
+        if (navigationOptions.size() == 0) {
+            NavigationOption childNavigationOption = new NavigationOption(R.mipmap.sidemenu_children,
```

Comment:
This is in relation to the `mipmap`s created for the sidemenu icons. The app work OK and the icons render OK, does this have a rendering effect on other devices? Having non app-icons in the `mipmap` folders

Reply:
I haven't seen the same problem with these icons but I will definitely have an eye out for the same.